pages.zh/common/base32.md

```diff
@@ -0,0 +1,19 @@
+# base32
+
+> base32加密解密.
+
+- 加密一个文件:
```

Comment:
The gTranslate says you used the verb `encrypt`. This is not encryption because you don't need a passphrase to get the original message.

Reply:
You're right. I used my own idiom and now I know I've been using the wrong word  😅